src/Factory/RestControllerFactory.php

```diff
@@ -143,7 +143,6 @@ public function createServiceWithName(ServiceLocatorInterface $controllers, $nam
             ));
         }
 
-        $controller->setEventManager($events);
```

Comment:
@kevinpapst It was definitely a bug. Additionally, [our own documentation indicates that resource listeners should listen to `ZF\Rest\Resource`](https://apigility.org/documentation/modules/zf-rest#zfrestabstractresourcelistener).